### ISSUE TITLE
avoid continuous wal archiving

### ISSUE THREAD

**Issue by fvannee** [pgbackrest/pgbackrest#1031]
I've been reading the documentation, but couldn't find a clear yes/no for my question. It seems like it's not possible, but I wanted to double-check.

In a case where a) I don't care about PITR and b) I want to avoid continuous streaming of WAL segments to a repository host. Is it possible to only do the archive-push of WAL segments to the repository host while I'm actively running a backup?

**Comment by dwsteele** [comment]
We don't support this directly yet but there is a fairly simple solution detailed in issue #900.

If you do this, *do not* disable the `archive-check` option. When `archive-check` is enabled (the default) the WAL will be checked after each backup to ensure nothing has gone wrong with your archive command.

**Comment by fvannee** [comment]
Thanks! That makes sense. 

One more thing, this may be a bug or something I'm completely overlooking on my side. I set up pgbackrest (master) on two nodes under my own user account. Passwordless ssh access between hosts is fine.

Repo:
```
[demo]
pg1-host=pg-server
pg1-path=/home/florisvannee/data/pg1
pg1-socket-path=/tmp
pg1-user=postgres
pg1-host-user=florisvannee

[global]
repo1-path=/home/florisvannee/pg/repo1
repo1-retention-full=2
start-fast=y
log-path=/home/florisvannee/pg/log
```

Postgres:
```
[demo]
pg1-path=/home/florisvannee/data/pg1
pg1-socket-path=/tmp
pg1-host-user=florisvannee
pg1-user=postgres

[global]
repo1-path=/home/florisvannee/pg/repo1
log-level-file=detail
log-path=/home/florisvannee/pg/log
repo1-host=repository-host
repo1-host-user=florisvannee
```

Now I can run the `check` and `backup` commands on the repository hosts. I've verified that the files are there and the backup looks good.

However, I cannot run a `check` or `restore` on the Postgres host:

```
pgbackrest --config=/home/florisvannee/pg/pgbackrest.conf --stanza=demo check
```

That command is OK on repository host, but outputs the following when ran on Postgres host:

```
ERROR: [037]: remote-0 process on '10.231.42.119' terminated unexpectedly [37]: ERROR: [037]: restore command requires option: pg1-path
              HINT: does this stanza exist?
```

This looks like a weird error to me as the `pg1-path` is set correctly in both cases and the stanza exists. Manually appending the `pg1-path` option to the pgbackrest command doesn't help. Any ideas what could be wrong here?

**Comment by dwsteele** [comment]
This is a known issue which we are planning to fix for 2.27. pgBackRest expects to find a config file on the repo host but doesn't because of your custom config locations.  If the config was in the standard location or if you specified the repo config location with `repo1-host-config` it should work: https://pgbackrest.org/configuration.html#section-repository/option-repo-host-config.

But as you say, there's enough info on the pg host to get the job done, so it should just be passed over and work. That's what were are doing for 2.27.

**Comment by fvannee** [comment]
Thanks, that works! Will close this issue :)
